src/test/unit/AVSDirectoryUnit.t.sol

```diff
@@ -170,14 +172,137 @@ contract AVSDirectoryUnitTests is EigenLayerUnitTestSetup, IAVSDirectoryEvents {
     }
 }
 
+contract AVSDirectoryUnitTests_registerOperatorToOperatorSet is AVSDirectoryUnitTests {
+    event OperatorAddedToOperatorSet(address operator, IAVSDirectory.OperatorSet operatorSet);
+
+    function testFuzz_revert_SignatureIsExpired(
+        address operator,
+        uint32 oid,
+        bytes32 salt,
+        uint256 expiry
+    ) public virtual {
```

Comment:
might want to add `filterFuzzedAddressInputs(operator)`

Reply:
+1 for all fuzzed address inputs, we get a couple flakey fuzz tests in our CI occasionally when we forget this modifier